web/src/app/users/UserEditDialog.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import { gql, useMutation } from '@apollo/client'
+import Spinner from '../loading/components/Spinner'
+import FormDialog from '../dialogs/FormDialog'
+import { useSessionInfo } from '../util/RequireConfig'
+import { Checkbox, FormControlLabel } from '@material-ui/core'
```

Comment:
nit, move material import above local files:
```suggestion
import { gql, useMutation } from '@apollo/client'
import { Checkbox, FormControlLabel } from '@material-ui/core'
import Spinner from '../loading/components/Spinner'
import FormDialog from '../dialogs/FormDialog'
import { useSessionInfo } from '../util/RequireConfig'
```